examples/zcAbsent_scenario.hs

```diff
@@ -20,7 +20,7 @@ circuit uv oc zc gp_ack gn_ack gp gn =
     gpHandshake = handshake gp gp_ack
     gnHandshake = handshake gn gn_ack
 
-    initialState = initialise0 [uv, oc, zc, gp_ack, gn_ack, gp, gn]
+    initialState = initialise0 [uv, oc, zc, gp_ack, gp] <> initialise1 [gn_ack, gn]
 
```

Comment:
I beleive it's like this in the exampoles in the article . Shall I change those too?